config/application.rb

```diff
@@ -1,50 +1,46 @@
-require File.expand_path('../boot', __FILE__)
+require_relative "boot"
 
 require "rails"
 # Pick the frameworks you want:
-require "active_storage/engine"
 require "active_model/railtie"
 require "active_job/railtie"
 require "active_record/railtie"
+require "active_storage/engine"
 require "action_controller/railtie"
 require "action_mailer/railtie"
+require "action_mailbox/engine"
+require "action_text/engine"
 require "action_view/railtie"
+# require "action_cable/engine"
 require "sprockets/railtie"
-# require "rails/test_unit/railtie"
+require "rails/test_unit/railtie"
 
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
 module TmdbMoviequeue
   class Application < Rails::Application
-     # Initialize configuration defaults for originally generated Rails version.
-     config.load_defaults 5.2
-    # Settings in config/environments/* take precedence over those specified here.
-    # Application configuration can go into files in config/initializers
-    # -- all .rb files in that directory are automatically loaded after loading
-    # the framework and any gems in your application.
-
-    # Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
-    # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
-    # config.time_zone = 'Central Time (US & Canada)'
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 6.0
 
-    # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
-    # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
-    # config.i18n.default_locale = :de
+    ActiveSupport::Inflector.inflections(:en) do |inflect|
+      inflect.acronym 'TV'
+    end
+    # Configuration for the application, engines, and railties goes here.
+    #
+    # These settings can be overridden in specific environments using the files
+    # in config/environments, which are processed later.
     config.generators do |g|
-        g.test_framework :rspec,
-        fixtures: true,
-        view_specs: false,
-        helper_specs: false,
-        routing_specs: false,
-        controller_specs: true,
-        request_specs: false
-        g.fixture_replacement :factory_girl, dir: "spec/factories"
+      g.test_framework :rspec,
+      fixtures: true,
+      view_specs: false,
+      helper_specs: false,
+      routing_specs: false,
+      controller_specs: true,
+      request_specs: false
+      g.fixture_replacement :factory_bot, dir: "spec/factories"
     end
-
-    # Do not swallow errors in after_commit/after_rollback callbacks.
-    config.autoload_paths += %W(#{config.root}/lib)
-    config.time_zone = 'Central Time (US & Canada)'
```

Comment:
Do we want to set our time zone? Do you know what happens to our user experience if we don't?

Reply:
Ah, yeah I forget the impacts but I added it back. I didn't mean to remove it 😅